src/gl-state-egl.cpp

```diff
@@ -293,7 +293,7 @@ EglConfig::print() const
 bool
 GLStateEGL::init_display(void* native_display, GLVisualConfig& visual_config)
 {
-    native_display_ = reinterpret_cast<EGLNativeDisplayType>(native_display);
+    native_display_ = static_cast<EGLNativeDisplayType>((intptr_t)native_display);
```

Comment:
What is the purpose of this casting change? On systems where EGLNativeDisplayType is a pointer type (e.g. X11), the static cast from intptr_t to EGLNativeDisplayType will fail. reinterpret_cast should work in all cases. Is there something special in QNX requiring this change?